hw1/TasksForHomework/Task1.java

```diff
@@ -0,0 +1,28 @@
+//package TasksForHomework;
+
+import java.util.Scanner;
+
+public class Task1 {
+
+    public static void main(String[] args) {
+
+        Scanner s1 = new Scanner(System.in);
+        System.out.println("Enter ur dogs age(years):");
+        int ageYears = s1.nextInt();
+
+        Scanner s2 = new Scanner(System.in);
+        System.out.println("Enter ur dogs age(months):");
+        int ageMonths = s2.nextInt();

```

Comment:
this variables hasn't used for logic... what if user writer 15 months (it is 1 year and 3 months)... don't forget about user can write anything